### ISSUE TITLE
Cygwin Error

### ISSUE THREAD

**Issue by Deathlive** [oh-my-fish/oh-my-fish#569]
```
CYGWIN_NT-10.0 GOD 2.9.0(0.318/5/3) i686
~/.local/share/omf/themes/bobthefish/fish_greeting.fish (line 3):
  uptime
  ^
in function “fish_greeting”
        called on line 141 of file /usr/share/fish/functions/__fish_config_interactive.fish

in function “__fish_config_interactive”
        called on line 210 of file /usr/share/fish/config.fish

in function “__fish_on_interactive”
        called on standard input

in event handler: handler for generic event “fish_prompt”
```
I install bobthefish theme.
How to fix this ?

**Comment by oranja** [comment]
Cygwin is not really a supported platform for `fish` or `omf`, but this seems like an easy one.

The way `fish` reports unhandled and unexpected errors is through a stack-trace. The line that actually generated the error is at the top. Below it is a hierarchy of functions, each one called by the function below, which gives some hints on how we got to the state of error.

Here at the top we can see that the error occurred right when trying to call a utility program called `uptime`. It's very common on *nix distributions, but on a Cygwin setup it's reasonable to guess that it's simply not installed.

Using the `cygcheck` utility or Cygwin's [Package Search](https://cygwin.com/cgi-bin2/package-grep.cgi) webpage, we can find out which package provides this utility. In this case, you can find `procps-ng` in the list of packages that provide `uptime`, and the description seems to match. It's reasonable that a package of "System and process monitoring utilities" will provide a utility called `uptime` that will tell how long the system has been up and running.

Bottom line: Try to install the package `procps-ng` for Cygwin.

Hope this solves your issue and helps you in the future.

**Comment by sagebind** [comment]
Closing due to inactivity. If you continue to experience similar Cygwin issues feel free to reopen or to open a new case. 😀
